barcode-scanning/ios/BarcodeScanning.m

```diff
@@ -9,10 +9,8 @@ @implementation BarcodeScanning
 
 RCT_EXPORT_MODULE()
 
-RCT_EXPORT_METHOD(scan
-                  : (nonnull NSString *)url resolver
-                  : (RCTPromiseResolveBlock)resolve rejecter
-                  : (RCTPromiseRejectBlock)reject) {
+RCT_EXPORT_METHOD(scan : (nonnull NSString *)url resolver : (
+    RCTPromiseResolveBlock)resolve rejecter : (RCTPromiseRejectBlock)reject) {
```

Comment:
Nit: The formatting looks odd to me (the previous one was odd too 😅). I'd prefer something like this.
```suggestion
RCT_EXPORT_METHOD(scan:(nonnull NSString *)url
                  resolver:(RCTPromiseResolveBlock)resolve
                  rejecter:(RCTPromiseRejectBlock)reject) {
```